### ISSUE TITLE
Read access exception

### ISSUE THREAD

**Issue by thomas-klambauer** [jshiell/checkstyle-idea#450]
Got this exceptino in checkstyle idea 5.29.0 and IDEA 2019.1 Ultimate on linux

```
Details: Current thread: Thread[ApplicationImpl pooled thread 684,4,Idea Thread Group] 1664753506
; dispatch thread: false; isDispatchThread(): false
SystemEventQueueThread: Thread[AWT-EventQueue-0 2019.1.3#IU-191.7479.19 IDEA, eap:false, os:Linux 4.9.0-9-amd64, java-version:JetBrains s.r.o 1.8.0_202-release-1483-b58,6,Idea Thread Group] 2080259553

java.lang.Throwable: Read access is allowed from event dispatch thread or inside read-action only (see com.intellij.openapi.application.Application.runReadAction())
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:162)
	at com.intellij.openapi.application.impl.ApplicationImpl.assertReadAccessAllowed(ApplicationImpl.java:1087)
	at com.intellij.openapi.module.impl.ModuleManagerImpl.moduleDependencyComparator(ModuleManagerImpl.java:572)
	at com.intellij.openapi.module.ModuleUtilCore.findModuleForPsiElement(ModuleUtilCore.java:107)
	at org.infernus.idea.checkstyle.checker.ScanFiles.mapsModulesToFiles(ScanFiles.java:63)
	at org.infernus.idea.checkstyle.checker.ScanFiles.<init>(ScanFiles.java:48)
	at org.infernus.idea.checkstyle.CheckStylePlugin.scanFiles(CheckStylePlugin.java:211)
	at org.infernus.idea.checkstyle.handlers.ScanFilesBeforeCheckinHandler$2.run(ScanFilesBeforeCheckinHandler.java:96)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:731)
	at com.intellij.openapi.progress.impl.CoreProgressManager$5.run(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:586)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:86)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$null$10(ApplicationImpl.java:594)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:311)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by jshiell** [comment]
Thanks for the report. Looks like the semantics of `ModuleUtil.findModuleForPsiElement(PsiFile)` have changed - I'll update this to wrap it as a read action.

**Comment by jshiell** [comment]
Should be fixed in 5.29.2.

**Comment by thomas-klambauer** [comment]
Cool,thanks!
